src/data-catalog/create-dataset/CreateDataset.ts

```diff
@@ -4,7 +4,7 @@ import { div, h } from 'react-hyperscript-helpers';
 import { ButtonPrimary, spinnerOverlay } from 'src/components/common';
 import FooterWrapper from 'src/components/FooterWrapper';
 import { TopBar } from 'src/components/TopBar';
-import { Ajax } from 'src/libs/ajax';
+import { Catalog } from 'src/libs/ajax/Catalog';
```

Comment:
Since `Catalog` is also imported from below, sonar flags this as a case where imports can be merged? Can you check and address sonar warnings added with this PR?

Reply:
ah, missed that.  will do